src/services/videoconference/index.js

```diff
@@ -36,326 +35,316 @@ const {
 	CREATE_OPTION_TOGGLES,
 } = require('./logic/constants');
 
+const CLIENT_HOST = Configuration.get('HOST');
+
 const VideoconferenceModel = require('./model');
 const { schoolModel: Schools } = require('../school/model');
 
 const { ObjectId } = require('../../helper/compare');
 
-const CLIENT_HOST = Configuration.get('HOST');
+// event ids are from postgres instead of mongo
+function scopeIdMatchesEventId(id) {
```

Comment:
move  `scopeIdMatchesEventId` to https://github.com/schul-cloud/schulcloud-server/blob/develop/src/helper/compare.js within a new WhateverPostgresId?

Reply:
To be honest, I don't really get what this function is supposed to handle. It looks like it's just supposed to check the `id` is a valid UUID string (which it doesn't really do well looking at the RegExp). 🤔 

It's not clear here what this has to do with Postgres. Can you enlighten me so I can maybe add a code comment or make another small update? Thanks 🙂